src/y2start/y2start

```diff
@@ -48,16 +48,13 @@ NO_CLI_CLIENTS = ["installation", "view_anymsg", "firstboot", "scc"].freeze
 Yast.ui_create(args[:server_name], args[:server_options])
 # set application title bsc#1033161
 Yast.import "UI"
-# do not fail if gethostname failed
-hostname = Socket.gethostname rescue ""
-hostname = "" if hostname == "(none)"
-hostname = " @ #{hostname}" unless hostname.empty?
+Yast.import "Arch"
```

Comment:
I am not sure if this is good idea to start more depending in starting script on YaST itself. I vote in this case for some easier and more straighforward way to detect s390.
I know here it import UI, but only to do initial initialization of title and we already have a lot of problems with it ( see below ).

Reply:
What are you suggesting ?

Reply:
something like calling `arch` and check output?

Reply:
I will take the "arch" call instead. Is it OK to use Yast::Execute.on_target at least ? ;-)

Reply:
I see it also depend on YaST. So I will change it too.

Reply:
well, Execute is basically cheetah with some flavor and flavor is mostly logger. So it can cause issue if we want to tune a bit logger in this script. Potential issue is to use `on_target` without `!` which can open popup.